Containerfile.zfs

```diff
@@ -0,0 +1,42 @@
+###
+### Containerfile.common - used to build ONLY NON-nvidia kmods
+###
+
+ARG FEDORA_MAJOR_VERSION="${FEDORA_MAJOR_VERSION:-40}"
+ARG KERNEL_FLAVOR="${KERNEL_FLAVOR:-coreos-stable}"
+ARG KERNEL_IMAGE="${KERNEL_IMAGE:-${KERNEL_FLAVOR}-kernel}"
+ARG KERNEL_ORG="${KERNEL_ORG:-ublue-os}"
+ARG KERNEL_BASE="ghcr.io/${KERNEL_ORG}/${KERNEL_IMAGE}:${FEDORA_MAJOR_VERSION}"
+ARG BUILDER_IMAGE="${BUILDER_IMAGE:-quay.io/fedora/fedora}"
+ARG BUILDER_BASE="${BUILDER_IMAGE}:${FEDORA_MAJOR_VERSION}"
+FROM ${KERNEL_BASE} AS kernel_cache
+FROM ${BUILDER_BASE} AS builder
+
+ARG FEDORA_MAJOR_VERSION="${FEDORA_MAJOR_VERSION:-40}"
+ARG KERNEL_FLAVOR="${KERNEL_FLAVOR:-coreos-stable}"
+ARG RPMFUSION_MIRROR=""
+ARG ZFS_MINOR_VERSION="${ZFS_MINOR_VERSION:-2.2}"
+
+COPY build*.sh /tmp
+COPY certs /tmp/certs
+
+# cached kernel rpms
+COPY --from=kernel_cache /tmp/rpms /tmp/kernel_cache
+
+# Set kernel name
+RUN if grep -q "coreos" <<< "${KERNEL_FLAVOR}"; then \
+        /tmp/build-prep.sh && \
+        /tmp/build-kmod-zfs.sh \
+    ; else \
+        echo "Error: ublue only builds ZFS for CoreOS kernels." \
+    ; fi
+
+RUN for RPM in $(find /var/cache/akmods/ -type f -name \*.rpm); do \
```

Comment:
We were placing in a subdirectory since this also builds debug and other packages.

Reply:
This is the same logic which we have in `ucore-kmods/Containerfile` and `akmods/Containerfile.common` ... 

It will result in the ZFS RPMs eventually being on the scratch image as:
- `/rpms/zfs/` - runtime RPMs
- `/rpms/zfs/other/` - test and dracut RPMs
- `/rpms/zfs/devel/` - devel RPMs
- `/rpms/zfs/debug/` - debug RPMs
- `/rpms/zfs/src/` - source RPMs


This seems good to me... not to you? are you wanting to strip the `zfs` from `/rpms/zfs/`?

Reply:
Nope we're good then